README.md

```diff
@@ -3332,6 +3332,185 @@ These fields are optionally available:
 
 
 
+### wp signup
+
+Manages signups on a multisite installation.
+
+~~~
+wp signup
+~~~
+
+**EXAMPLES**
+
+    # List signups.
+    $ wp signup list
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+    | signup_id | user_login | user_email          | registered          | active | activation_key   |
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+    | 1         | bobuser    | bobuser@example.com | 2024-03-13 05:46:53 | 1      | 7320b2f009266618 |
+    | 2         | johndoe    | johndoe@example.com | 2024-03-13 06:24:44 | 0      | 9068d859186cd0b5 |
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+
+    # Activate signup.
+    $ wp signup activate 2
+    Success: Signup activated. Password: bZFSGsfzb9xs
+
+    # Delete signup.
+    $ wp signup delete 3
+    Success: Signup deleted.
+
+
+
+### wp signup activate
+
+Activates a signup.
+
+~~~
+wp signup activate <signup>
+~~~
+
+**OPTIONS**
+
+	<signup>
+		Signup ID, user login, user email or activation key.
+
+**EXAMPLES**
+
+    # Activate signup.
+    $ wp signup activate 2
+    Success: Signup activated. Password: bZFSGsfzb9xs
+
+
+
+### wp signup delete
+
+Deletes a signup.
+
+~~~
+wp signup delete <signup>
+~~~
+
+**OPTIONS**
+
+	<signup>
+		Signup ID, user login, user email or activation key.
+
+**EXAMPLES**
+
+    # Delete signup.
+    $ wp signup delete 3
+    Success: Signup deleted.
+
+
+
+### wp signup get
+
+Gets details about the signup.
+
+~~~
+wp signup get <signup> [--field=<field>] [--fields=<fields>] [--format=<format>]
+~~~
+
+**OPTIONS**
+
+	<signup>
+		Signup ID, user login, user email or activation key.
+
+	[--field=<field>]
+		Instead of returning the whole signup, returns the value of a single field.
+
+	[--fields=<fields>]
+		Get a specific subset of the signup's fields.
+
+	[--format=<format>]
+		Render output in a particular format.
+		---
+		default: table
+		options:
+		  - table
+		  - csv
+		  - json
+		  - yaml
+		---
+
+**EXAMPLES**
+
+    # Get signup.
+    $ wp signup get 1
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+    | signup_id | user_login | user_email          | registered          | active | activation_key   |
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+    | 1         | bobuser    | bobuser@example.com | 2024-03-13 05:46:53 | 1      | 663b5af63dd930fd |
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+
+
+
+### wp signup list
+
+Lists signups.
+
+~~~
+wp signup list [--field=<field>] [--<field>=<value>] [--fields=<fields>] [--format=<format>]
+~~~
+
+	[--field=<field>]
+		Prints the value of a single field for each signup.
+
+	[--<field>=<value>]
+		Filter results by key=value pairs.
```

Comment:
The de facto description appears to be "Filter the list by a specific field."; see [this search](https://github.com/search?q=repo%3Awp-cli%2Fentity-command+%22%5B--%3Cfield%3E%3D%3Cvalue%3E%5D%22&type=code).
```suggestion
		Filter the list by a specific field.
```
Also, `[--<field>=<value>]` is usually first in the list.